docs/project-setup-wizard.md

```
@@ -0,0 +1,268 @@
+| Gradle project name | `--name` |
+| Application ID | `--application-id` |
+| Launcher display name | `--display-name` |
+| Convention-plugin alias | `--plugin-alias` |
+| Author | `--author` |
+| Version code and name | `--version-code`, `--version-name` |
+| Android SDK levels | `--compile-sdk`, `--min-sdk`, `--target-sdk`, `--benchmark-min-sdk` |
+| Retained posts-example backend | `--posts-backend-url` |
+| Starter content | `--remove-examples` or `--keep-examples`; `--ai-free` or `--include-ai` |
+| Curated dependencies | `--preset`; repeat `--capability` for additions |
+| Output | `--archive`, `--output`, or `--in-place` |
+| Final checks | `--format`, `--verify none|narrow|full` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)